solutions/remotemonitoring/single-vm/setup.sh

```diff
@@ -72,6 +72,26 @@ export APPLICATION_SECRET=$PCS_APPLICATION_SECRET
 
 # ========================================================================
 
+### Install Docker
+
+install_docker_ce() {
+    apt-get update
+    # Remove old packages if installed
+    set +e
+    apt-get remove docker docker-engine docker.io
+    set -e
+    # Install Docker's GPG key
+    apt-get -y --force-yes --no-install-recommends install apt-transport-https ca-certificates curl gnupg2 software-properties-common
+    curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | sudo apt-key add -
+    # Add Docker repository to get up to date packages
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable"
```

Comment:
If we don't use mirror for China cloud, it might fail Mooncake deployment. The mirror site is: https://mirror.azure.cn/docker-ce